src/ripple/app/ledger/Ledger.cpp

```diff
@@ -942,6 +942,8 @@ static bool saveValidatedLedger (
                 JLOG (j.warn())
                     << "Transaction in ledger " << seq
                     << " affects no accounts";
```

Comment:
I guess if you don't actually need a distinct log record for this JSON, you could just add it to the line above with a "\n" since they are at the same level. Looks fine to me either way.